extension/app/ts/background/windows/confirmTransaction.ts

```diff
@@ -30,34 +29,44 @@ const onCloseWindow = () => { // check if user has closed the window on their ow
 }
 
 export async function openConfirmTransactionDialog(
-	port: browser.runtime.Port,
-	request: InterceptedRequest,
+	requestId: number,
+	origin: string,
 	simulationMode: boolean,
 	transactionToSimulatePromise: () => Promise<EthereumUnsignedTransaction>,
 ) {
-	if (window.interceptor.settings === undefined) return
-	if (port.sender?.url === undefined) return
+	if (window.interceptor.settings === undefined) {
+		return {
+			error: {
+				code: 1,
+				message: 'Interceptor not ready'
+			}
+		}
+	}
```

Comment:
I see this error at least twice.  Should it be extracted into a library of "common errors" or something both to ensure we don't repeat ourselves and to ensure we get consistently shaped errors delivered?

Reply:
I added couple of these to constants. We could look to see how to improve these more in to future thought